component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRowStructVisitor.java

```diff
@@ -502,6 +514,9 @@ private Entry toEntry(final String name, final Schema.Type type, final String or
             props.put(PATTERN, pattern);
         }
         props.put(STUDIO_TYPE, studioType);
+        if (logicalType != null) {
```

Comment:
Please use withLogicalType in the "return factory..." don' put it in a temporary map taht will be used in withProps(...)

```
       return factory
                .newEntryBuilder()
                .withName(name)
                .withRawName(originalName)
                .withNullable(isNullable)
                .withType(type)
                .withComment(comment)
                .withDefaultValue(defaultValue)
                .withProps(props)
                .withLogicalType(SchemaProperty.LogicalType.valueOf(logicalType))
                .build();
```